common/src/cellular_pktio.c

```diff
@@ -911,7 +908,7 @@ static void _handleAllReceived( CellularContext_t * pContext,
     while( keepProcess == true )
     {
         /* Pktio is reading command. Skip over the change line. */
-        while( ( ( *pTempLine == '\r' ) || ( *pTempLine == '\n' ) ) && ( bytesRead > 0U ) )
+        while( ( *pTempLine == '\r' ) || ( *pTempLine == '\n' ) )
```

Comment:
This won't happen because the caller of this function has bounded the input already.
https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/blob/e9da3fe417f6cad36ac8fc63ad19135c33d836aa/common/src/cellular_pktio.c#L982

Reply:
[This commit](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/26/commits/a5ff0c94eddd3b1f1d12b9296265dffa2c5c92a6) resolve the comment.